src/app/policies/views/PolicyListView.vue

```diff
@@ -219,8 +235,30 @@ const change = function (e: PolicyCollection) {
   color: var(--grey-400);
 }
 
-.entity-heading {
-  font-size: inherit;
-  font-weight: var(--font-weight-regular);
+.toolbar {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: var(--spacing-md);
+  padding: var(--spacing-sm) var(--spacing-sm) 0 var(--spacing-sm);
+
+}
```

Comment:
**Suggestion**: These styles shouldn’t be own a user of AppCollection but rather AppCollection itself.

Can we remove them in favor of https://github.com/kumahq/kuma-gui/pull/1203?

Reply:
Yes exactly 👍

Reply:
Ok this is also now done.